src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DBFilterBase.java

```diff
@@ -288,13 +285,85 @@ boolean canAddField() {
       }
     }
   }
+
+  /**
+   * Filters db documents based on non document id field values if id use "$in" operator, use
+   * IDFilter, since partition key 'or' is not supported besides id, "$in" operator, use INFilter
+   */
+  public static class INFilter extends DBFilterBase {
```

Comment:
Capitalization should be `InFilter` (otherwise it'd be read as... "infinity filter"? :) )

Reply:
solved